src/middle/Stan_math_signatures.ml

```diff
@@ -103,6 +103,14 @@ let rec ints_to_real unsized =
   | UArray t -> UArray (ints_to_real t)
   | x -> x
 
+let rec complex_to_real = function
```

Comment:
Can we put this in UnsizedType?

Reply:
We could but I don't think we should. Like the function above it (`ints_to_real`) I think the only safe uses for it in the compiler are in writing signatures. If you can think of other places we would want it I can move it over